src/resolve/resolved-dns-query.c

```diff
@@ -585,6 +588,16 @@ void dns_query_complete(DnsQuery *q, DnsTransactionState state) {
 
         q->state = state;
 
+        if (state == DNS_TRANSACTION_SUCCESS && set_size(q->manager->varlink_subscription) > 0) {
+                DnsQuestion *question = q->request_packet ? q->request_packet->question : NULL;
+                const char *query_name = question ? dns_question_first_name(question) : q->request_name;
+                if (query_name) {
+                        r = send_dns_notification(q->manager, q->answer, query_name);
+                        if (r < 0)
+                                log_debug_errno(r, "Failed to send varlink notification, ignoring: %m");
```

Comment:
All errors in send_dns_notification() are already logged. Please drop the log here, and voidify the function call.

Reply:
Done